astropy/analytic_functions/blackbody.py

```diff
@@ -66,7 +66,7 @@ def blackbody_nu(in_x, temperature):
     # Check if input values are physically possible
     if temp < 0:
         raise ValueError('Invalid temperature {0}'.format(temp))
-    if np.any(freq <= 0):  # pragma: no cover
+    if not np.all(np.isfinite(freq)) or np.any(freq <= 0):  # pragma: no cover
```

Comment:
It is probably because I did not write a test for it, so I excluded it from coverage report. If there is a test now, you can remove the `pragma`.